libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -236,6 +237,15 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
     }
   }
 
+  private listenForIonModalWillPresent() {
+    if (this.ionModalElement) {
+      this.ionModalElement.addEventListener('ionModalWillPresent', () => {
+        if (this.config.interactWithBackground)
+          this.renderer.addClass(document.body, 'allow-scroll');
```

Comment:
Please use `windowRef` instead:
```suggestion
          this.renderer.addClass(this.windowRef.document.body, 'allow-scroll');
```

Reply:
Fixed in next commit